types/telegram-web-app/index.d.ts

```diff
@@ -505,6 +531,100 @@ interface HapticFeedback {
+     * will contain the error. In case of success, the first argument will be
+     * null and the second argument will be a boolean indicating whether the
+     * value was stored.
+     */
+    setItem(
+        key: string, value: string,
+        callback?: (error: Error | null, success?: boolean) => void,
+    ): CloudStorage;
+    /**
+     * A method that receives a value from the cloud storage using
+     * the specified key.
+     *
+     * @param key The key should contain 1-128 characters, only A-Z, a-z, 0-9,
+     * _ and - are allowed.
+     * @param callback In case of an error, the callback function will
+     * be called and the first argument will contain the error. In case of
+     * success, the first argument will be null and the value will be passed
+     * as the second argument.
+     */
+    getItem(
+      key: string,
+      callback?: (error: Error | null, value?: string) => void,
+    ): CloudStorage;
+    /**
+     * A method that receives values from the cloud storage using the specified
+     * keys.
+     *
+     * @param key The keys should contain 1-128 characters, only A-Z, a-z, 0-9,
+     * _ and - are allowed.
+     * @param callback In case of an error, the callback? function will be
+     * called and the first argument will contain the error. In case of
+     * success, the first argument will be null and the values will be passed
+     * as the second argument.
+     */
+    getItems(
+      keys: string[],
+      callback?: (error: Error | null, values?: string[]) => void,
+    ): CloudStorage;
+    /**
+     * A method that removes a value from the cloud storage using the specified
+     * key.
+     *
+     * @param key The key should contain 1-128 characters, only A-Z, a-z, 0-9,
+     * _ and - are allowed.
+     * @param callback If an optional callback parameter was passed, the
+     * callback function will be called. In case of an error, the first
+     * argument will contain the error. In case of success, the first
+     * argument will be null and the second argument will be a boolean
+     * indicating whether the value was removed.
+     */
+    removeItem(
+      key: string,
+      callback?: (error: Error | null, success: boolean) => void,
+    ): CloudStorage;
+    /**
+     * A method that removes values from the cloud storage using the specified
+     * keys.
+     *
+     * @param key The keys should contain 1-128 characters, only A-Z, a-z, 0-9,
+     * _ and - are allowed.
+     * @param callback If an optional callback parameter was passed, the
+     * callback function will be called. In case of an error, the first
+     * argument will contain the error. In case of success, the first
+     * argument will be null and the second argument will be a boolean
+     * indicating whether the values were removed.
+     */
+    removeItems(
+      keys: string[],
+      callback?: (error: Error | null, success: boolean) => void,
```

Comment:
`success` contains `true` in case of success and `null` in case of an error
```suggestion
      callback?: (error: Error | null, success: boolean | null) => void,
```